charts/hail-search/templates/deployment.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
     {{- include "hail-search.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  strategy:
```

Comment:
in case anyone sees this... I'm investigating cluster autoscaling as an option here, so that we don't have to be limited by static nodes during our deploy.  

this code is an option, basically saying our deploy won't allow anything beyond the `replicaCount` when doing the rolling deploy.  however, I think it's better that we scale up to `2 * replicaCount` when we deploy.